src/components/Display.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ result }) => (
+  <div className="display">
+    { result }
+  </div>
+);
+
```

Comment:
- [ ] The default value of the result should be 0. 
![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)

To define default prop value you need to use `defaultProps` property on the component. Kindly check [this](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) for more.